src/brogue/Rogue.h

```diff
@@ -3255,6 +3255,7 @@ extern "C" {
     void updatePlayerRegenerationDelay();
     boolean removeItemFromChain(item *theItem, item *theChain);
     void addItemToChain(item *theItem, item *theChain);
+    void increaseMaxHealthAndHeal(creature *monst, int maxHealthIncrease);
```

Comment:
I don't think this is necessary if `increaseMaxHealthAndHeal` is moved above the first usage in Items.c.  If done, the function can also be changed to `static void`.